### DISCUSSION TITLE
Added: LDM Notes for February 24th, 2021

### DISCUSSION THREAD

**Discussion by 333fred** [dotnet/csharplang#4470] [LDM Notes]
https://github.com/dotnet/csharplang/blob/master/meetings/2021/LDM-2021-02-24.md

## Agenda

1. Static abstract members in interfaces

**Comment by MgSam** [comment]
Is there a proposal yet for the math interfaces? Will that be in dotnet/runtime?

**Reply by 333fred** [reply]
It will be in the runtime repo.
